src/DetailsView/components/settings-panel/settings-panel.scss

```diff
@@ -3,7 +3,7 @@
 @import '../../../common/styles/colors.scss';
 
 .settings-panel {
-    .generic-toggle-component .toggle-description {
+    :global(.toggle-description) {
```

Comment:
Dumb question: is it actually desirable/necessary for the description text color to be secondary-text in the settings panel but the default primary text color in the preview features panel (the only other place where generic toggle appears to be used)? I'd have expected us to want those to look consistent, in which case this probably just belongs in `generic-toggle.scss`

Reply:
Good point. Honestly, I was following on what we had before (as far as I know) but it does make sense and I check with Fer too.